### PR TITLE
change fastmode to fastscan in dev-guide

### PR DESCRIPTION
**Pull request by joey-yez**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Change fastmode to FastScan in dev-guide, and modify TOC

We decided to change the name of the feature fastmode to fast scan.
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 1]
```diff
@@ -7,20 +7,21 @@ summary: An overview of the usage of ALTER TABLE ... SET TIFLASH MODE ... for th
 
 > **Warning:**
 >
-> This statement is still an experimental feature. It is NOT recommended that you use it in the production environment.
+> This statement is still an experimental feature.
```
Comment (hfxsd) [inline sql-statements/sql-statement-set-tiflash-mode.md:10]:
```suggestion
> This statement is still an experimental feature. It is NOT recommended that you use it in the production environment.
```

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 2]
```diff
@@ -8,19 +8,20 @@ summary: An overview of the usage of ALTER TABLE ... SET TIFLASH MODE ... for th
 > **Warning:**
 >
 > This statement is still an experimental feature. It is NOT recommended that you use it in the production environment.
```
Comment (hfxsd) [inline sql-statements/sql-statement-set-tiflash-mode.md:10]:
```suggestion
> This statement is still an experimental feature.
```

File: develop/dev-guide-use-fastscan.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+> This feature is still an experimental feature. 
+> The form and usage of related experimental features may change in subsequent versions.
+
+This document describes how to use FastScan to speed up queries in Online Analytical Processing (OLAP) scenarios.
+
+By default, TiFlash guarantees the precision of query results and data consistency. With the feature FastScan, TiFlash provides more efficient query performance, but does not guarantee the accuracy of query results and data consistency.
+
+Some OLAP scenarios allow for some tolerance to the accuracy of the query results. In these cases, if you need higher query performance, you can enable FastScan for the corresponding table for querying.
+
+The FastScan takes effect globally only for those tables who enables FastScan by [ALTER TABLE SET TIFLASH MODE](/sql-statements/sql-statement-set-tiflash-mode.md). TiFlash-related operations are not supported for temporary tables, in-memory tables, system tables, and tables with non-utf-8 characters in the column names, and therefore enabling FastScan is not supported for them.
```
Comment (hfxsd) [inline develop/dev-guide-use-fastscan.md:19]:
```suggestion
The FastScan takes effect globally only for those tables for which you enable FastScan by running [ALTER TABLE SET TIFLASH MODE](/sql-statements/sql-statement-set-tiflash-mode.md). TiFlash-related operations are not supported for temporary tables, in-memory tables, system tables, and tables with non-utf-8 characters in the column names, and therefore enabling FastScan is not supported for them.
```

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 4]
```diff
@@ -7,20 +7,21 @@ summary: An overview of the usage of ALTER TABLE ... SET TIFLASH MODE ... for th
...
+> This statement is still an experimental feature.
+> The form and usage of related experimental features may change in subsequent versions.
 
-You can use the `ALTER TABLE...SET TIFLASH MODE...` statement to switch the mode of the corresponding table in TiFlash. The following modes are currently supported.
+You can use the `ALTER TABLE...SET TIFLASH MODE...` statement to switch the FastScan option for the corresponding table in TiFlash. The following options are currently supported.
 
-- `Normal Mode`. The default mode. This mode guarantees the accuracy of query results and data consistency.
-- `Fast Mode`. This mode does not guarantee the accuracy of query results and data consistency, but provides more efficient query performance.
+- `Normal Mode`. The default option, means disabling FastScan. In this option, TiFlash guarantees the accuracy of query results and data consistency.
+- `Fast Mode`. The FastScan option, means enableing FastScan. In this option, TiFlash provides more efficient query performance, but does not guarantee the accuracy of query results and data consistency.
```
Comment (shichun-0415) [inline sql-statements/sql-statement-set-tiflash-mode.md:16]:
```suggestion
- `Normal Mode`. The default option. In this mode, FastScan is disabled and the accuracy of query results and data consistency are guaranteed.
- `Fast Mode`. In this mode, FastScan is enabled. TiFlash provides more efficient query performance, but does not guarantee the accuracy of query results and data consistency.
```

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 5]
```diff
@@ -7,20 +7,21 @@ summary: An overview of the usage of ALTER TABLE ... SET TIFLASH MODE ... for th
...
 
-This statement executes without blocking the execution of existing SQL statements or the running of TiDB features, such as transactions, DDL, and GC, and without changing the data content accessed through the SQL statement. The statement will end normally when the mode switch is completed.
+This statement executes without blocking the execution of existing SQL statements or the running of TiDB features, such as transactions, DDL, and GC, and without changing the data content accessed through the SQL statement. The statement will end normally when the option switch is completed.
```
Comment (shichun-0415) [inline sql-statements/sql-statement-set-tiflash-mode.md:18]:
```suggestion
The execution of `ALTER TABLE ... SET TIFLASH MODE ...` does not block existing SQL statements or interrupt other TiDB features, such as transactions, DDL, and GC. At the same time, data accessed through SQL statements is not altered. This statement ends normally when the mode is switched.
```

File: develop/dev-guide-use-fastscan.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+The data in the storage layer of TiFlash is stored in two layers: Delta layer and Stable layer.
+
+The overall TableScan algorithm process in Normal Mode consists of the following steps:
+
+1. Read data: separate data streams are created in the Delta layer and Stable layer to read the respective data.
+2. Sort Merge: the data streams created in Step 1 are sorted and merged, and the data is returned in (handle, version) order.
+3. Range Filter: according to the data range, filter the data generated in Step 2, and then return the data.
+4. MVCC + Column Filter: filter the data generated in step 3 through MVCC. Meanwhile, filter out the unneeded columns, and then return the data.
+
+FastScan gains faster query performance by sacrificing some data consistency. The FastScan omits Step 2 and the MVCC part in Step 4 in the normal scan process, thus improving query performance.
```
Comment (shichun-0415) [inline develop/dev-guide-use-fastscan.md:56]:
```suggestion
FastScan gains faster query speed by sacrificing some data consistency. Step 2 and the MVCC part in step 4 in the normal scan process are omitted in FastScan, thus improving query performance.
```

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 7]
```diff
@@ -7,20 +7,21 @@ summary: An overview of the usage of ALTER TABLE ... SET TIFLASH MODE ... for th
...
 
-This statement only supports changing the mode to tables in TiFlash, so the mode change only affects reads involving the TiFlash table.
+This statement only supports switching the FastScan option for tables in TiFlash, so the switch only affects reads involving the TiFlash table.
```
Comment (shichun-0415) [inline sql-statements/sql-statement-set-tiflash-mode.md:20]:
```suggestion
You can only execute this statement to switch the FastScan option for tables in TiFlash. Therefore, option change only affects reading tables in TiFlash.
```

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 8]
```diff
@@ -7,20 +7,21 @@ summary: An overview of the usage of ALTER TABLE ... SET TIFLASH MODE ... for th
...
 
-The mode change to tables in TiFlash takes effect only if the table has a TiFlash Replica. If the TiFlash Replica of the table is empty when you change the mode, the mode will take effect only after the TiFlash Replica is subsequently reset. You can use [`ALTER TABLE ... SET TIFLASH REPLICA ...`](/sql-statements/sql-statement-alter-table.md) to reset the TiFlash Replica.
+The FastScan switch takes effect only if the table has a TiFlash Replica. If the TiFlash Replica of the table is empty when you switch the option, the option will take effect only after the TiFlash Replica is subsequently reset. You can use [`ALTER TABLE ... SET TIFLASH REPLICA ...`](/sql-statements/sql-statement-alter-table.md) to reset the TiFlash Replica.
```
Comment (shichun-0415) [inline sql-statements/sql-statement-set-tiflash-mode.md:22]:
```suggestion
The FastScan switch takes effect only if the table has TiFlash replicas. If no TiFlash replica is present when you switch the option, the option takes effect only after TiFlash replicas are configured. You can use [`ALTER TABLE ... SET TIFLASH REPLICA ...`](/sql-statements/sql-statement-alter-table.md) to configure TiFlash replicas.
```

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 9]
```diff
@@ -53,7 +54,7 @@ SELECT table_mode FROM information_schema.tiflash_replica WHERE table_name = 'te
 +------------+
 ```
 
-If you want to enable Fast Mode to query the `test` table, execute the following statement to switch the mode, and you can query the mode of the current table.
+If you want to enable FastScan to query the `test` table, execute the following statement to switch the option, and you can query the option of the current table.
```
Comment (shichun-0415) [inline sql-statements/sql-statement-set-tiflash-mode.md:57]:
```suggestion
To enable FastScan for the `test` table, execute the following statement and then check whether FastScan is enabled for this table.
```

File: sql-statements/sql-statement-set-tiflash-mode.md [hunk 10]
```diff
@@ -68,7 +69,7 @@ SELECT table_mode FROM information_schema.tiflash_replica WHERE table_name = 'te
 +------------+
 ```
 
-If you want to switch back to Normal Mode mode, execute the following statement to switch.
+If you want to disable FastScan, execute the following statement to switch.
```
Comment (shichun-0415) [inline sql-statements/sql-statement-set-tiflash-mode.md:72]:
```suggestion
To disable FastScan, execute the following statement.
```

File: develop/dev-guide-use-fastscan.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+> This feature is still an experimental feature. 
+> The form and usage of related experimental features may change in subsequent versions.
+
+This document describes how to use FastScan to speed up queries in Online Analytical Processing (OLAP) scenarios.
+
+By default, TiFlash guarantees the precision of query results and data consistency. With the feature FastScan, TiFlash provides more efficient query performance, but does not guarantee the accuracy of query results and data consistency.
+
+Some OLAP scenarios allow for some tolerance to the accuracy of the query results. In these cases, if you need higher query performance, you can enable FastScan for the corresponding table for querying.
+
+The FastScan takes effect globally only for those tables for which you enable FastScan by running [ALTER TABLE SET TIFLASH MODE](/sql-statements/sql-statement-set-tiflash-mode.md). TiFlash-related operations are not supported for temporary tables, in-memory tables, system tables, and tables with non-utf-8 characters in the column names, and therefore enabling FastScan is not supported for them.
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply develop/dev-guide-use-fastscan.md:19]: 删掉了

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**